integration_tests/e2e/overview/goal/goalsPage.cy.ts

```diff
@@ -6,7 +6,7 @@ context('View goals', () => {
 
   beforeEach(() => {
     cy.task('reset')
-    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubSignInAsUserWithViewAuthority')
```

Comment:
For all of these cypress tests, where we are saying its now a readonly page where you dont need any roles, I think we should change this to:
```
cy.task('stubSignIn')
```

In addition, I think you want to make an edit to `cypress.config.ts`:
```
    setupNodeEvents(on) {
      on('task', {
        reset: resetStubs,
        ...auth,

        stubSignIn: (roles: []) => auth.stubSignIn(roles),
        stubSignInAsUserWithEditAuthority: () => auth.stubSignIn(['ROLE_EDUCATION_WORK_PLAN_EDITOR']),
        stubSignInAsUserWithViewAuthority: () => auth.stubSignIn(['ROLE_EDUCATION_WORK_PLAN_VIEWER']),
```
and remove the `stubSignInAsUserWithViewAuthority` setup altogether, because we are basically saying that that role no longer exists any more 👍